portfolio/src/main/java/com/google/sps/servlets/SubmitEmailServlet.java

```diff
@@ -19,12 +26,26 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     // Get the value entered in the form.
-    String email = request.getParameter("email-address");
+    // String email = request.getParameter("email-address");
```

Comment:
If not needed, can you remove this line?

Reply:
Yes, I removed this line